### PR TITLE
docs: add a bit more about spatial data

### PR DESCRIPTION
**Pull request by maelle**
WIP. TODOS

- [ ] Align introductory vignette with README

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -26,6 +26,7 @@ options(timeout = 2000)
 
 The goal of emodnet.wfs is to allow interrogation of and access to [EMODnet's, European Marine Observation and Data Network, geographic vector data](https://emodnet.ec.europa.eu/en/emodnet-web-service-documentation#inline-nav-3) in R through the [EMODnet Web Feature Services](https://emodnet.ec.europa.eu/en/data-0). 
 [Web Feature services (WFS)](https://www.ogc.org/publications/standard/wfs/) represent a change in the way geographic information is created, modified and exchanged on the Internet and offer direct fine-grained access to geographic information at the feature and feature property level. 
+Features are _observations_ and feature properties are _variables_.
```
Comment (annakrystalli) [inline README.Rmd:29]:
```suggestion
Features are representation of geographic entities, such as a coastlines, marine protected areas, offshore platforms, or fishing areas. In WFS, features have geometry (spatial information) and attributes (descriptive data).
```

File: README.Rmd [hunk 2]
```diff
@@ -59,7 +71,7 @@ services$service_name
 
 To explore available services you can use `View()` or your usual way to explore `data.frames`.
 
-## Create Service Client
+## Connect to a data source: Create a service client
```
Comment (annakrystalli) [inline README.Rmd:74]:
```suggestion
## Initialise a WFS Service Client

A WFS service client is responsible for sending requests to a WFS server and processing the responses to retrieve, display, or analyze geospatial features. As such, initialising a client is the first step to interacting with an EMODnet Web Feature Services.
```

File: README.Rmd [hunk 3]
```diff
@@ -69,9 +81,9 @@ wfs_bio <- emodnet_init_wfs_client(service = "biology")
 wfs_bio
 ```
 
-## Get WFS Layer info
+## List contents of a data source: Get layer information from a service client
```
Comment (annakrystalli) [inline README.Rmd:84]:
```suggestion
## List contents of a WFS: Get layer information from a service client

In the context of a Web Feature Service (WFS), a layer refers to a logical grouping of geographic features that share the same schema (i.e., the same feature type, geometry, and attributes). Layers are the units of data that clients can query, retrieve, and manipulate through a WFS.
```

File: README.Rmd [hunk 4]
```diff
@@ -69,9 +81,9 @@ wfs_bio <- emodnet_init_wfs_client(service = "biology")
...
 
-You can get metadata about the layers available from a service. 
+You can get metadata about the datasets, called layers, available from a service.
```
Comment (annakrystalli) [inline README.Rmd:86]:
```suggestion
You can access information (metadata) about each layer available from an EMODnet WFS with `emodnet_get_wfs_info()`
```

File: README.Rmd [hunk 5]
```diff
@@ -99,16 +111,20 @@ emodnet_get_all_wfs_info()
...
+
+By default, `emodnet_get_layers()` returns a list of sf objects, one per layer.
+
 
 ```{r}
 emodnet_get_layers(wfs = wfs_bio, layers = layers)
 ```
 
-You can change the output `crs` through the argument `crs`.
+You can change the output `crs` ("projection") through the argument `crs`.
```
Comment (annakrystalli) [inline README.Rmd:127]:
```suggestion
You can change the output Coordinate Reference System (CRS), which defines how geographic data is mapped to the Earth's surface,  through the argument `crs`.
```

CRS's can have projections (if they are projected) but you can also have geographic CRS's which are not projected, so projection is not quite accurate.

**Review by annakrystalli — COMMENTED** [review]
Thanks for taking a shot at this @maelle !

I've made some suggestions to keep the language closer to WFS terminology and added some extra definitions that hopefully make it easier for folks unfamiliar with WFS to follow what's going on. It's a bit of a divergence to what we discussed last week (sorry!), but on revisiting and adding some definitions, it seemed more accurate and clear.

File: README.Rmd (hunk 5 above)
Comment (maelle) [reply README.Rmd:127]: I should read the basic resources myself

File: vignettes/emodnet.wfs.Rmd [hunk 6]
```diff
@@ -1,13 +1,20 @@
 ---
 title: "Accessing and mapping EMODnet data"
 output: rmarkdown::html_vignette
+df_print: "tibble"
```
Comment (maelle) [inline vignettes/emodnet.wfs.Rmd:4]: OMG this is so elegant and simple

**Comment by maelle** [comment]
@annakrystalli thank you!!

@salvafern thank you!! I will port your edit to the .orig file and then knit it.

File: vignettes/emodnet.wfs.Rmd (hunk 6 above)
Comment (salvafern) [reply vignettes/emodnet.wfs.Rmd:4]: I know right? And there are more options: https://pkgs.rstudio.com/rmarkdown/reference/output_format.html#arguments-1

**Review by salvafern — APPROVED** [review]

**Pull request merged**